cmd/buildah/rmi.go

```diff
@@ -159,7 +159,19 @@ func rmiCmd(c *cli.Context) error {
 				}
 			}
 
-			if len(image.Names) > 0 {
+			isParent, err := imageIsParent(store, image.TopLayer)
+			if err != nil {
+				return err
+			}
+			if len(image.Names) > 0 || isParent {
```

Comment:
should this 'if' be moved to line 170?  I don't think 170 can ever be hit if isParent is true now.

Reply:
Changed the condition so that we don't try to delete a parent image that has been untagged and an image that still has a name associated with it after untagging it.